aiogram/types/message.py

```diff
@@ -228,22 +228,23 @@ def html_text(self) -> str:
         return self.parse_entities()
 
     @property
-    def url(self) -> str:
+    def url(self, always_private=False) -> str:
```

Comment:
@MasterGroosha, `url` is a `property`, so having any parameter except `self` in its getter is impossible. descriptor's getter has to have the following signature `(self, instance, owner) -> T` :)

You may take a look at https://docs.python.org/3/howto/descriptor.html and remember descriptors.

Reply:
Oh, haven't noticed. :eyes: